com.slugchess.SlugChess.yml

```diff
@@ -0,0 +1,59 @@
+app-id: com.slugchess.SlugChess
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: '/app/bin/slugchess-run.sh'
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options: 
+  append-path: '/usr/lib/sdk/dotnet7/bin'
+  append-ld-library-path: '/usr/lib/sdk/dotnet7/lib'
+  env: 
+    PKG_CONFIG_PATH: '/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig'
+    DOTNET_CLI_TELEMETRY_OPTOUT: 'true'
+  arch:
+#    arm: 
+#      env: 
+#        RUNTIME: linux-arm
+#    aarch64: 
+#      env: 
+#        RUNTIME: linux-arm64
+    x86_64: 
+      env: 
+        RUNTIME: linux-x64
+modules:
+  - name: flatpak-dotnet
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p /app/bin'
+      - 'dotnet publish -c Release -r $RUNTIME --source nuget-sources --no-self-contained -p:PublishSingleFile=false   SlugChess/SlugChess.csproj'
+      - 'cp -r --remove-destination /run/build/flatpak-dotnet/SlugChess/bin/Release/net7.0/$RUNTIME/publish/* /app/bin/'
+      - 'cp -r /run/build/flatpak-dotnet/flatpak/slugchess-run.sh /app/bin/'
+      # add icon
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - 'cp /run/build/flatpak-dotnet/SlugChess/Assets/images/svg/sc-black.svg /app/share/icons/hicolor/scalable/apps/com.slugchess.SlugChess.svg'
+      # add .desktop
+      - 'install -Dm 644 -t /app/share/applications flatpak/com.slugchess.SlugChess.desktop'
+      # add manifest
+      #- 'install -Dm 644 -t /app/share/metainfo flatpak/com.slugchess.SlugChess.metainfo.xml'
+      # temp fix for com.slugchess.SlugChess.appdata.xml in repo not having screenshots
+      - 'install -Dm 644 -t /app/share/metainfo com.slugchess.SlugChess.metainfo.xml'
+      - 'install.sh'
+    sources: 
+      - main-sources.json
+      - sources-extra.json
+      - slugchess-sources.json
+      # temp fix for com.slugchess.SlugChess.appdata.xml in repo not having screenshots
+      - type: file
+        path: com.slugchess.SlugChess.metainfo.xml
+finish-args:
+  # X11
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
if it supports wayland then it's `fallback-x11` on line 51.

Reply:
I have committed this change